cli/command/service/create.go

```diff
@@ -58,6 +59,9 @@ func newCreateCommand(dockerCli command.Cli) *cobra.Command {
 	flags.Var(&opts.hosts, flagHost, "Set one or more custom host-to-IP mappings (host:ip)")
 	flags.SetAnnotation(flagHost, "version", []string{"1.25"})
 
+	flags.Var(cliopts.NewListOptsRef(&opts.resources.resGenericResources, ValidateSingleGenericResource), "generic-resources", "User defined resources request")
```

Comment:
I think we generally use the singular form for flags, even with multiple values are supported, so this should be `generic-resource` as the flag name.

How about "User defined resource" for the help text?

Reply:
Thanks for noticing this